.circleci/config.yml

```diff
@@ -44,7 +38,9 @@ jobs:
       - checkout
       - run:
           name: Run tests
-          command: python3 setup.py test -P visualization --remote-data=astropy --open-files -a "--mpl --mpl-results-path=$PWD/results -W ignore:np.asscalar"
+          command: |
+            pip install -e .[test] pytest-mpl
+            pytest astropy/visualization --remote-data=astropy --open-files --mpl --mpl-results-path=$PWD/results -W ignore:np.asscalar
```

Comment:
I think these tests (and the ones below) now no longer include the documentation. For visualization, that seems a bad idea... (this goes back to @bsipocz's comment that the one single thing to keep from `setup.py test` is the `-P` option!)

Reply:
As promised, I've now written up a small pytest plugin to add the ``-P`` option:

https://github.com/astropy/pytest-filter-subpackage

I've updated this PR to use this - for now I'm installing the plugin from the git repository, but before this PR is merged, I'll do a release and add it as a pytest-astropy dependency.

Reply:
Super!!

Reply:
I've now made a pytest-astropy release, and am about to push changes to the CI to use this, so resolving this.